source/help/getting-started/managing-members.rst

```diff
@@ -39,7 +69,27 @@ Adding Someone to Your Team
 
 If someone you want to add to your team already has an account on the server, you can add them to your team without sending an invitation.
 
-**To add someone to your team**:
+**To add someone to your team**
```

Comment:
Given the steps here and in the direct invite are similar, I'm wondering if there is benefit of combining them vs having in two separate sections?

Reply:
We had some feedback that it was a bit confusing in the UI.  I'd prefer to keep them separate until the invite flow has been updated to make adding more obvious in 5.18.